src/button/styles.scss

```diff
@@ -165,3 +164,11 @@
     inset-inline: 0;
   }
 }
+
+.link.disabled {
+  pointer-events: none;
+}
+
+.disabled-reason-tooltip {
+  /* used in test-utils or tests */
```

Comment:
Let's extract the classes used in test utils in a separate file (see #2303) to ensure we've stable test-util selectors.

Reply:
good catch. updated